src/main/java/org/fcrepo/transform/transformations/LDPathTransform.java

```diff
@@ -78,60 +74,57 @@ public LDPathTransform(final InputStream query) {
         this.query = query;
     }
 
-    /**
-     * Pull a node-type specific transform out of JCR
-     * @param node the node
-     * @param key the key
-     * @return node-type specific transform
-     * @throws RepositoryException if repository exception occurred
-     */
-    public static LDPathTransform getNodeTypeTransform(final Node node,
-        final String key) throws RepositoryException {
-
-        final Node programNode = node.getSession().getNode(CONFIGURATION_FOLDER + key);
+    public static LDPathTransform getResourceTransform(final FedoraResource resource, final NodeService nodeService,
```

Comment:
During the extraction of fcrepo-transform from the core codebase the checkstyle plugin was dropped. Please add a javadoc to this method in advance of that plugin being reintroduced.